ios/Utils/RNSVGPathMeasure.m

```diff
@@ -85,10 +75,18 @@ void subdivideBezierAtT(const CGPoint bez[4], CGPoint bez1[4], CGPoint bez2[4],
 }
 
 @implementation RNSVGPathMeasure
+/**
+* returns the distance between two points
+*/
++(CGFloat) distanceFrom:(CGPoint) p1 to:(CGPoint) p2 {
```

Comment:
Could you also add the above function `subdivideBezierAtT` as a class method, analogous to what you did with `distance`? I believe we can merge it then.

Reply:
After consultation with @kmagiera, it should be ever easier to not move these methods to class, but just provide `static` keyword at the beginning of them, which will result in not including them in global namespace. Could you do it @notjulian ?

Reply:
@WoLewicki I've made the changes as you and @kmagiera suggested ...

Reply:
OK, I will change the target branch for this PR since now `main` is the default one and we will see if there are any conflicts, because there are right now for some reason

Reply:
Hmm the conflict is still there 😕  Could you merge the current `main` branch into your branch to see what is causing the issue exactly and resolve it? Otherwise I cannot merge it.

Reply:
can you please check now?

Reply:
It is ok now :tada: if the CI passes, it's ok to be merged.